app/models/maestrano/connector/rails/external.rb

```diff
@@ -6,15 +6,14 @@ def self.external_name
   end
 
   def self.get_client(organization)
-    BaseCRM::Client.new access_token: organization.oauth_token
+    BaseAPIManager.new(organization)
   end
 
   def self.entities_list
-    %w(person_and_organization item)
+    %w(person_and_organization item opportunity app_user)
```

Comment:
We'll need to order them logically, with those with the least reference first
So app_user should be first